### ISSUE TITLE
[MAS2.4.7] Keyboard focus indicator is not visible at 'Welcome', and 'Live chat' push buttons given as different tabs.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-Emulator#715]
**Actual Result:**
Keyboard focus indicator is not visible at 'Welcome', 'Settings' and 'Live chat' push buttons given as different tabs.

**Expected Result:**
Keyboard focus indicator should be properly visible at all tab buttons.

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Click on open bot and choose the .bot file that is available in the bot folder.
3. Open Settings and Welcome screen also.
4. Notice 'Welcome', 'Settings' and 'Live chat' push buttons given as different tabs.
5. Navigate to these tab buttons and notice that keyboard tab focus is not visible at these tab buttons.

**User Impact:**
Keyboard focus indicator is important for keyboard dependent users to locate exact position of keyboard focus.

**Test Environment:**
OS: Windows 10
OS Build: 17134.167
OS Version: 1803
Browser: Edge

**Impacted MAS Link:**
[MAS 2.4.7 – Focus Visible (05A)(https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c})

**Attachment:**
[MAS2.4.7_BotFrameworkEmulator_TabButtons.zip](https://github.com/Microsoft/BotFramework-Emulator/files/2245268/MAS2.4.7_BotFrameworkEmulator_TabButtons.zip)

**Comment by corinagum** [comment]
Reopening - I noticed that on latest that the Close tabs button (x) are no longer keyboard accessible.
